.github/labeler.yml

```diff
@@ -0,0 +1,83 @@
+# Add 'enhancement' label to any PR where the head branch name starts with `feat/`
+enhancement:
+  - head-branch: ['^feat/']
+
+# Add 'bug' label to any PR where the head branch name starts with `fix/`
+bug:
+  - head-branch: ['^fix/']
+
+# Add 'build' label to any PR where the head branch name starts with `build/`
+build:
+  - head-branch: ['^build/']
+  - changed-files:
+    - any-glob-to-any-file:
+      - '.nvmrc'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'rollup.config.mjs'
+      - 'vite.config.js'
+
+# Add 'ci' label to any PR where the head branch name starts with `ci/`
+github_actions:
```

Comment:
This label is currently used to identify bump of gh actions only. PR related to CI changes are labeled with "core".
Labels are used to generate the release notes and until now, we didn't consider to split CI changes from other "chore" changes